README.md

```diff
@@ -1134,6 +1134,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Domain
   * [Freenom](https://freenom.com) — Free .tk,.ml,.ga,.cf and .gq domain.
+  * [Free Domains](https://freesubdomains.org) - Free subdomains on multiple supported domains for personal sites, open-source projects, and more.
```

Comment:
Sorry we do not accept services like this

Reply:
Why? There are others like this.

Reply:
You are welcome to send a PR to remove all the little cloudflare based GitHub repos you can find here. But new ones will not be added.